src/main/java/edu/ucsb/cs156/rec/controllers/DummyController.java

```diff
@@ -0,0 +1,26 @@
+package edu.ucsb.cs156.rec.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.ucsb.cs156.rec.errors.EntityNotFoundException;
+
+
+/**
+ * This class is used to test ApiController and EntityNotFoundException
+ */
+
+@RequestMapping("/dummycontroller")
+@RestController
+public class DummyController extends ApiController {
+
+    @GetMapping("")
+    public String getById(@RequestParam Long id) throws EntityNotFoundException {
+        if (id == 1) {
+            return "String1";
+        }
+        throw new EntityNotFoundException(String.class, id);
+    }
+}
```

Comment:
This file should be placed under `src/test/java/edu/ucsb/cs156/rec/controllers/DummyController.java` not `src/main/java/edu/ucsb/cs156/rec/controllers/DummyController.java`